pom.xml

```diff
@@ -894,7 +895,7 @@
       <dependency>
         <groupId>javax.annotation</groupId>
         <artifactId>javax.annotation-api</artifactId>
-        <version>1.2</version>
+        <version>1.3</version>
```

Comment:
@janbartel should we stay with javax.annotation at 1.2 for jetty 9.4.x?

Reply:
the idea was to fix ``` [INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-java) @ jetty-http-spi ---
[WARNING] Rule 6: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for javax.annotation:javax.annotation-api:1.2 paths to dependency are:
+-org.eclipse.jetty:jetty-http-spi:9.4.14-SNAPSHOT
  +-com.sun.xml.ws:jaxws-rt:2.3.0.2
    +-javax.annotation:javax.annotation-api:1.2 (managed) <-- javax.annotation:javax.annotation-api:1.3 ```
it's not really important as it's used for testing purpose.